core/sr-api-macros/src/lib.rs

```diff
@@ -78,6 +78,15 @@ mod utils;
 ///
 /// /// All runtime api implementations need to be done in one call of the macro!
 /// impl_runtime_apis! {
+/// #   impl client::runtime_api::Core<Block> for Runtime {
```

Comment:
I'm curious what you're fixing here.

Reply:
All runtime apis require that the `Core` api is implemented. I really don't know why this worked before. The CI was also not complaining without this, but locally it failed :man_shrugging: 

Basically its a doc test that shows the user how to use the macro.